docs/postgres.md

```diff
@@ -0,0 +1,40 @@
+# Running dcrpool with PostgreSQL
```

Comment:
I think this could use a db optimization section,  check  https://pgtune.leopard.in.ua/#/ your local setup to see what can be configuration options improve the db setup.

Reply:
Done